### PR TITLE
DOC: fill in missing docstrings for Series.sample

### PR DESCRIPTION
**Pull request by natmokval**
xref #71

### CODE REVIEW

**Comment by DeaMariaLeon** [comment]
Thanks @natmokval, I left a comment..

**Comment by natmokval** [comment]
> Thanks @natmokval, I left a comment..

thanks for your comment

File: narwhals/series.py [hunk 1]
```diff
@@ -697,6 +697,49 @@ def sample(
...
+            >>> import pandas as pd
+            >>> import polars as pl
+
+            >>> s_pd = pd.Series([1, 2, 3, 4])
+            >>> s_pl = pl.Series([1, 2, 3, 4])
+
+            We define a library agnostic function:
+
+            >>> def func(s_any):
+            ...     nw.from_native(s_any, series_only=True)
```
[the remark this answers is not in the document]
Comment (natmokval) [reply narwhals/series.py]: updated

**Comment by DeaMariaLeon** [comment]
I wonder if we should add a note explaining that it's OK to have different results. 
Or if we should use a 'seed' to have the same results. 
Friendly ping to @MarcoGorelli.. ?

**Review by MarcoGorelli — COMMENTED** [review]
thanks for doing this!

yeah a little note saying not to expect consistent results across libraries (due to the nature of the method) would probably be good

I don't think they'd respect seeds the same way anyway

**Comment by natmokval** [comment]
thanks @MarcoGorelli, @DeaMariaLeon for helping me with this PR. I added a note about consistency in the behavior of `Series.sample`

**Review by MarcoGorelli — APPROVED** [review]
thanks @natmokval !

**Pull request merged**